### ISSUE TITLE
Milestone 1 Feedback

### ISSUE THREAD

**Issue by yuliaUU** [STAT547-UBC-2019-20/group_04_anh_eve#11]
- You did a very good job analyzing and introducing your dataset! and thank you for providing proper references and having a solid introduction that illustrates what is the issue is and why you decided to analyze your dataset!
- Data exploration is great!
- minor issue: *Looking at the correlations of the pollutants with weather..* you forgot to put a line break between a graph and a text, so this line appears in the wrong place. Make sure you spend more time checking your knitted document to avoid these minor mistakes.
- suggestion: it will help to read your report easier if you write your data exploration in terms of a story: first by stating what you want to explore, how, provide results and then conclude, and transition to the next analysis. otherwise, it reads 'jumpy'.
- minor issue: your header is *Research question* and you did not ask any. maybe rephrase you goal into a question or change the title 
- Plan of action is great and I am curious what you will obtain in your future analysis!

very awesome job!!

**Comment by evelynjulia** [comment]
Thanks for the feedback! We changed the research question to actually be a question and I think we fixed the line break issue!
